### ISSUE TITLE
Example code in README.md is incorrect

### ISSUE THREAD

**Issue by yukenknk** [rylio/ytdl#56]
The last line in the example golang code is incorrect and does not work.
It should be something like the code below with the first argument being the file format

`vid.Download(vid.Formats[0],file)`

**Comment by muriithiderro** [comment]
Thanks alot for that, I also found this from test samples
```format := vid.Formats.Worst(ytdl.FormatResolutionKey)[0]```

**Comment by wedojava** [comment]
The correct example, I just  push to request readme update.
hope it can help guys.
```
package main

import (
	"fmt"
	"os"

	"github.com/rylio/ytdl"
	"github.com/rs/zerolog/log"
)

func main() {
	client := ytdl.Client{
        	HTTPClient: http.DefaultClient,
        	Logger:     log.Logger,
 	}
	vid, err := client.GetVideoInfo("https://www.youtube.com/watch?v=WkVvG4QTO9M")
	if err != nil {
		fmt.Println("Failed to get video info")
		return
	}
	file, _ := os.Create(vid.Title + ".mp4")
	defer file.Close()
	client.Download(vid, vid.Formats[0], file)
}
```

**Comment by corny** [comment]
solved by ee74b0402c7a91ad72c94e0ef5af4729292e8c25
